crud/update.go

```diff
@@ -0,0 +1,62 @@
+package crud
+
+import (
+	"context"
+
+	"github.com/tarantool/go-tarantool"
+)
+
+// UpdateRequest helps you to create request
+// object to call `crud.update` for execution
+// by a Connection.
+type UpdateRequest struct {
+	spaceRequest
+	key        interface{}
+	operations interface{}
```

Comment:
It's better to have a separate type for it like `Condition`.
```suggestion
	operations []Operation
```

Reply:
Done.